docs/sources/flow/reference/components/discovery.triton.md

```diff
@@ -0,0 +1,125 @@
+---
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/discovery.triton/
+title: discovery.triton
+---
+
+# discovery.triton
+
+`discovery.triton` discovers [Triton][] Container Monitors and exposes them as targets.
+
+[Triton]: https://www.tritondatacenter.com
+
+## Usage
+
+```river
+discovery.triton "LABEL" {
+	account    = ACCOUNT
+	dns_suffix = DNS_SUFFIX
+	endpoint   = ENDPOINT
+}
+```
+
+## Arguments
+
+The following arguments are supported:
+
+Name               | Type           | Description                                                         | Default       | Required
+------------------ | -------------- | ------------------------------------------------------------------- | ------------- | --------
+`account`          | `string`       | The account to use for discovering new targets.                     |               | yes
+`role`             | `string`       | The type of targets to discover.                                    | `"container"` | no
+`dns_suffix`       | `string`       | The DNS suffix which should be applied to target.                   |               | yes
+`endpoint`         | `string`       | The Triton discovery endpoint (e.g. 'cmon.us-east-3b.triton.zone'). |               | yes
+`groups`           | `list(string)` | A list of groups for which targets are retrieved.                   |               | no
```

Comment:
```suggestion
`groups`           | `list(string)` | A list of groups to retrieve targets from.                   |               | no
```

You can end a sentence with a preposition. This description is not wrong the way it was written but it's not conversational or natural. You can leave it or change it.. up to you.